tests/acceptance/features/apiSpaces/moveSpaces.feature

```diff
@@ -0,0 +1,326 @@
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+  Scenario Outline: Moving a file within same space project with role manager and editor
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has created a folder "newfolder" in space "Project"
+    And user "Brian" has uploaded a file inside space "Project" with content "some content" to "insideSpace.txt"
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    When user "Alice" moves file "insideSpace.txt" to "newfolder/insideSpace.txt" in space "Project" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Alice" the space "Project" should contain these entries:
+      | newfolder/insideSpace.txt       |
+    And for user "Alice" the space "Project" should not contain these entries:
+      | insideSpace.txt       |
+    Examples:
+      | role    |
+      | manager |
+      | editor  |
+
+
+  Scenario: Moving a file within same space project with role viewer
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has created a folder "newfolder" in space "Project"
+    And user "Brian" has uploaded a file inside space "Project" with content "some content" to "insideSpace.txt"
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    When user "Alice" moves file "insideSpace.txt" to "newfolder/insideSpace.txt" in space "Project" using the WebDAV API
+    Then the HTTP status code should be "403"
+    And for user "Alice" the space "Project" should not contain these entries:
+      | newfolder/insideSpace.txt       |
+    And for user "Alice" the space "Project" should contain these entries:
+      | insideSpace.txt       |
+
+
+  Scenario Outline: User move a file from a space project with different role to a space project with manager role
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project1" with the default quota using the GraphApi
+    And user "Brian" has created a space "Project2" with the default quota using the GraphApi
+    And user "Brian" has uploaded a file inside space "Project1" with content "Project1 content" to "project1.txt"
+    And user "Brian" has shared a space "Project2" to user "Alice" with role "manager"
+    And user "Brian" has shared a space "Project1" to user "Alice" with role "<role>"
+    When user "Alice" moves file "project1.txt" from space "Project1" to "project1.txt" inside space "Project2" using the WebDAV API
+    Then the HTTP status code should be "502"
```

Comment:
502 is server error, is this the expected status code?

Reply:
yes, that is expected. MOVE operations are not possible between two different spaces.

Reply:
@kiranparajuli589  https://github.com/owncloud/ocis/issues/4087#issuecomment-1177951438

Reply:
I thought it would be possible to perform move operation between spaces. If its not possible then i have to refactor the scenarios using the Example tables

Reply:
@micbar If it is not possible, shouldn't the response code should be 4xx with some error message in resp.? 5xx represents server error or crashes which is not the case because the server is doing fine.

Reply:
according to @butonic `502` is according to spec.

In my ideal world, the clients would not offer a  cross storage space move and inform the user that they can do a COPY & DELETE instead. :smile: